### PR TITLE
AF-2177 :Add Project button should also allow importing

### PR DESCRIPTION
**Pull request by RishiRajAnand**
- Removed the Kabab menu from the Space_Name header
- Added delete space button in place of kebab menu
- Added import project and try samples menu items in add project sidemenu.

### CODE REVIEW

**Comment by RishiRajAnand** [comment]
addprjectbutton
@paulovmr Let me know if its aligned to what we discussed :)

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/screens/PopulatedLibraryScreen.java [hunk 1]
```diff
@@ -114,13 +119,26 @@ public PopulatedLibraryScreen(final View view,
     public void setup() {
         view.init(this);
         if (userCanCreateProjects()) {
+            addProjectButtonPresenter.addOtherActions(this.translationService.getTranslation("TrySamples"), () -> trySamples());
+            addProjectButtonPresenter.addOtherActions(this.translationService.getTranslation("ImportProject"), () -> importProject());
```
[the remark this answers is not in the document]
Comment (RishiRajAnand) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/screens/PopulatedLibraryScreen.java]: Thanks @paulovmr for reviewing. Actually, the same component (AddProjectButton) is used in 2 places with different list items. So do you mean that we create a method in (in AddProjectButtonPresenter ) that adds "Try samples" and "Import button" items to the list and call the same from PopulatedLibraryScreen ?
addprjectbutton
noproject

[the remark this answers is not in the document]
Comment (paulovmr) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/screens/PopulatedLibraryScreen.java]: @RishiRajAnand Yes, better to have methods like `enableTrySamples()` and `enableImportProject()` in the AddProjectButtonPresenter.java that shows the hidden items.

[the remark this answers is not in the document]
Comment (RishiRajAnand) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/screens/PopulatedLibraryScreen.java]: Got it, Thanks!

**Comment by barboras7** [comment]
jenkins execute full downstream build

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/widgets/library/AddProjectButtonView.java [hunk 2]
```diff
@@ -57,6 +62,12 @@ public void addOtherProject(final MenuResourceHandlerWidget menuResourceHandlerW
         otherProjectsContainer.appendChild(menuResourceHandlerWidget.getElement());
     }
 
+    @Override
+    public void addNewMenuItem(final MenuResourceHandlerWidget menuResourceHandlerWidget) {
+        otherProjectsContainer.insertBefore(menuResourceHandlerWidget.getElement(), otherProjectsHeader);
```
Comment (paulovmr) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/widgets/library/AddProjectButtonView.java]: Aren't the items going between the divider and the header now?

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/widgets/library/AddProjectButtonPresenter.java [hunk 3]
```diff
@@ -115,6 +122,22 @@ public void addProject() {
         }
     }
 
+    public void enableTrySamples(){
+        final MenuResourceHandlerWidget menuResourceHandlerWidget = menuResourceHandlerWidgets.get();
+        menuResourceHandlerWidget.init(this.translationService.getTranslation("TrySamples"),
```
Comment (paulovmr) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/widgets/library/AddProjectButtonPresenter.java]: Could you extract the `"TrySamples"` string to a constant in `LibraryConstants.java`?

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/widgets/library/AddProjectButtonPresenter.java [hunk 4]
```diff
@@ -115,6 +122,22 @@ public void addProject() {
...
+                                       () -> libraryPlaces.closeAllPlacesOrNothing(() -> {
+                                               libraryPlaces.goToLibrary();
+                                               libraryPlaces.goToTrySamples();
+                                           }));
+        view.addNewMenuItem(menuResourceHandlerWidget);
+    }
+    public void enableImportProject() {
+        final MenuResourceHandlerWidget menuResourceHandlerWidget = menuResourceHandlerWidgets.get();
+        menuResourceHandlerWidget.init(this.translationService.getTranslation("ImportProject"),
```
Comment (paulovmr) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/widgets/library/AddProjectButtonPresenter.java]: Could you extract the `"ImportProject"` string to a constant in `LibraryConstants.java`?

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/widgets/library/AddProjectButtonView.html [hunk 5]
```diff
@@ -20,7 +20,7 @@
         <span class="caret"></span>
     </button>
     <ul class="dropdown-menu" data-field="other-projects-container">
-        <li class="dropdown-header" data-i18n-key="OtherProjects"></li>
+        <li class="dropdown-header" data-field="other-projects-header" data-i18n-key="OtherProjects"></li>
         <li class="divider"></li>
```
[the remark this answers is not in the document]
Comment (RishiRajAnand) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/widgets/library/AddProjectButtonView.html]: Yeah it was! Sorry i didn't understand it previous time. i think now its better aligned to patternfly dropdowns.
Screenshot from 2019-09-17 19-20-03

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/widgets/library/AddProjectButtonPresenter.java (hunk 4 above)
Comment (RishiRajAnand) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/widgets/library/AddProjectButtonPresenter.java]: Done! Thanks @paulovmr

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/widgets/library/AddProjectButtonPresenter.java (hunk 3 above)
Comment (RishiRajAnand) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/widgets/library/AddProjectButtonPresenter.java]: Done!

**Comment by dgutierr** [comment]
@RishiRajAnand mind to add Liz as a reviewer please and not merge under she approves?

**Comment by domhanak** [comment]
jenkins execute full downstream build

**Comment by lclay2** [comment]
UX accept/verified, but please note that we plan to revisit the use of combo button dropdown menus (versus kebab menus) in future.

**Comment by paulovmr** [comment]
@RishiRajAnand Looks like the selenium test failures are related to these changes. Could you check please?

**Comment by ederign** [comment]
please jenkins retest this

**Comment by ederign** [comment]
please jenkins retest this

**Comment by ederign** [comment]
please jenkins execute full downstream build

**Comment by ederign** [comment]
please jenkins execute full downstream build

**Comment by paulovmr** [comment]
jenkins execute full downstream build

**Comment by ederign** [comment]
jenkins execute full downstream build

**Comment by ederign** [comment]
please jenkins retest this

**Comment by ederign** [comment]
jenkins execute full downstream build

**Comment by ederign** [comment]
Test failures are not related:


Test Result (4 failures / ±0)org.kie.wb.selenium.ui.StandalonePerspectivesIntegrationTest.testStandaloneEditorPerspectiveorg.kie.wb.selenium.ui.StandalonePerspectivesIntegrationTest.testContentManagerPerspectiveorg.kie.wb.selenium.ui.StandalonePerspectivesIntegrationTest.testLibraryPerspectiveWithHeaderorg.kie.wb.selenium.ui.StandalonePerspectivesIntegrationTest.testLibraryPerspectiveWithoutHeader
--

**Pull request merged**